third_party/move/move-model/src/ast.rs

```diff
@@ -384,6 +384,18 @@ pub struct UseDecl {
     pub members: Vec<(Loc, Symbol, Option<Symbol>)>,
 }
 
+// =================================================================================================
+/// # Friend Declarations
+
+/// Represents a `friend` declaration in the source.
+#[derive(Debug, Clone)]
+pub struct FriendDecl {
+    /// Location covered by this declaration.
+    pub loc: Loc,
```

Comment:
Do we use this field anywhere?  I don't see it.

Reply:
Yes please keep this, its a public API which aims at providing all available info.